src/Widget/Menu.php

```diff
@@ -0,0 +1,791 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\DataView\Widget;
+
+use InvalidArgumentException;
+use ReflectionException;
+use Stringable;
+use Yiisoft\Html\Html;
+use Yiisoft\Html\Tag\CustomTag;
+use Yiisoft\Html\Tag\I;
+use Yiisoft\Html\Tag\Span;
+use Yiisoft\Widget\Widget;
+
+use function array_merge;
+use function count;
+use function implode;
+
+/**
+ * Menu displays a multi-level menu using nested HTML lists.
+ *
+ * The {@see Menu::items()} method specifies the possible items in the menu.
+ * A menu item can contain sub-items which specify the sub-menu under that menu item.
+ *
+ * Menu checks the current route and request parameters to toggle certain menu items with active state.
+ *
+ * Note that Menu only renders the HTML tags about the menu. It does not do any styling.
+ * You are responsible to provide CSS styles to make it look like a real menu.
+ *
+ * The following example shows how to use Menu:
+ *
+ * ```php
+ * <?= Menu::Widget()
+ *     ->items([
+ *         ['label' => 'Login', 'link' => 'site/login', 'visible' => true],
+ *     ]);
+ * ?>
+ * ```
+ */
+final class Menu extends Widget
```

Comment:
Doesn't seem to belong to this package.

Reply:
done move to https://github.com/yiisoft/yii-widgets/pull/53.